src/core_modules/capture-core/components/WidgetsRelationship/common/RelationshipsWidget/DeleteRelationship/DeleteRelationship.js

```diff
@@ -0,0 +1,82 @@
+// @flow
+
+import React, { useState } from 'react';
+import i18n from '@dhis2/d2-i18n';
+import {
+    DataTableCell,
+    IconDelete16,
+    Modal,
+    ModalContent,
+    ModalTitle,
+    ModalActions,
+    ButtonStrip,
+    Button,
+} from '@dhis2/ui';
+import { IconButton } from 'capture-ui';
+import { withStyles } from '@material-ui/core/styles';
+
+type Props = {
+    handleDeleteRelationship: () => void,
+    disabled?: boolean,
+    classes: {
+        tableCell: string,
+    },
+}
+
+const styles = {
+    tableCell: {
+        display: 'flex',
+        justifyContent: 'center',
+        zIndex: 1000,
+    },
+};
+
+export const DeleteRelationshipPlain = ({ handleDeleteRelationship, disabled, classes }: Props) => {
+    const [isModalOpen, setIsModalOpen] = useState(false);
+    return (
+        <>
+            <DataTableCell className={classes.tableCell}>
+                <IconButton
```

Comment:
Does it make sense to apply the destructive red color to the delete icon? Some suggestion:

<img width="388" alt="Screenshot 2024-01-31 at 14 56 58" src="https://github.com/dhis2/capture-app/assets/6817073/e76933d3-4d52-421f-8b99-c40d3e0ddc92">